### PR TITLE
fixed h tag height issue

### PR DESCRIPTION
**Pull request by ue**
### Issue number
#596

### CODE REVIEW

File: src/components/postElements/body/view/postBodyStyles.js [hunk 1]
```diff
@@ -14,6 +14,22 @@ export default EStyleSheet.create({
     color: '$primaryBlue',
     fontFamily: '$primaryFont',
   },
+  h4: {
+    fontSize: 15,
+    marginHorizontal: 10,
+    // height: 10,
```
Comment (mistikk) [inline src/components/postElements/body/view/postBodyStyles.js]: can you remove if not neccessary

File: src/components/postElements/body/view/postBodyStyles.js [hunk 2]
```diff
@@ -14,6 +14,22 @@ export default EStyleSheet.create({
...
+    marginBottom: 5,
+    marginTop: 5,
```
Comment (mistikk) [inline src/components/postElements/body/view/postBodyStyles.js]: you can change with margin vertical

File: src/components/postElements/body/view/postBodyStyles.js [hunk 3]
```diff
@@ -14,6 +14,22 @@ export default EStyleSheet.create({
...
+    fontSize: 30,
+  },
+  h2: {
+    fontSize: 25,
+  },
+  h3: {
+    fontSize: 20,
+  },
   img: {
     // height: 50,
```
Comment (mistikk) [inline src/components/postElements/body/view/postBodyStyles.js]: please remove

File: src/utils/markdownToHtml.js [hunk 4]
```diff
@@ -81,6 +82,10 @@ export const markDown2Html = (input) => {
     output = handleATag(output);
   }
 
+  if (hTagRegex.test(output)) {
+    output = handleHTag(output);
+  }
+
   // if (copiedPostRegex.test(output)) {
   //   output = handleMarkdownLink(output);
   // }
```
Comment (mistikk) [inline src/utils/markdownToHtml.js]: please remove

File: src/components/postElements/body/view/postBodyStyles.js (hunk 3 above)
Comment (ue) [reply src/components/postElements/body/view/postBodyStyles.js]: @mistikk comment out are necessary

File: src/utils/markdownToHtml.js (hunk 4 above)
Comment (ue) [reply src/utils/markdownToHtml.js]: these examples are necessary

**Pull request merged**